library/src/morpheus-master/sass/modules/navigation/_base.scss

```diff
@@ -485,6 +485,10 @@ body.is-logged-out{
 	{
 		text-decoration: none;
 
+		@media #{$phone} {
+			margin-top: 7px;
```

Comment:
Rather than adding a pixel value to push the text down arbitrarily, I would have preferred "display: flex" and "align-items: center" on the parent element: #mastLogin #loginLinks